goagen/codegen/helpers_test.go

```diff
@@ -0,0 +1,37 @@
+package codegen_test
+
+import (
+	"testing"
+
+	"github.com/goadesign/goa/goagen/codegen"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestHelpers(t *testing.T) {
+	var (
+		stringTestA = "testAa"
```

Comment:
I wanted to be specific but I have nothing against this suggestion :). I'll apply those changes